src/widgets/dropdown/lv_dropdown.c

```diff
@@ -142,7 +142,7 @@ const lv_obj_class_t lv_dropdownlist_class = {
     .event_cb = lv_dropdown_list_event,
     .instance_size = sizeof(lv_dropdown_list_t),
     .base_class = &lv_obj_class,
-    .name = "dropdown-list",
+    .name = "lv_dropdown-list",
```

Comment:
Should it be consistent with how you've removed hyphens i.e. `msgbox-content` to `lv_msgbox_content` etc.?

```suggestion
    .name = "lv_dropdown_list",
```

Reply:
It's intentional to make it consistent with how XML is looks like. There we have 
```xml
<lv_dropdown x="10", y="30">
    <lv_dropdown-list styles="red"/>
</lv_dropdown>
```